app/helpers/application_helper.rb

```diff
@@ -32,4 +32,9 @@ def assocation_empty?(assoc)
     assoc.reload unless assoc.nil? || assoc.loaded?
     assoc.nil? ? true : assoc.size == 0
   end
+
+  def i18n_time_ago_in_words(past_time)
+    "#{time_ago_in_words(past_time)} #{t('time_ago')}"
```

Comment:
Is this correct wording in Swedish (Susanna)?

If this works we're OK.  Generally, think it's better to pass in the `time_ago_in_words(past_time)` as a parameter to `t('time_ago')` in order to prevent "lego translation". (https://www.mediawiki.org/wiki/Localisation#Avoid_fragmented_or_.27patchwork.27_messages)